libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -50,4 +50,12 @@ + (NSString *)localizedString:(NSString *)localizationKey
     return NSLocalizedStringFromTableInBundle(localizationKey, @"Localizable", sdkBundle, nil);
 }
 
++ (UIImage *)imageNamed:(NSString *)name
+{
+    NSBundle *bundle = [NSBundle bundleWithIdentifier:@"com.salesforce.mobilesdk.SalesforceSDKCore"];
```

Comment:
We should stick with the paradigm used in `mainSdkBundle`: `[NSBundle bundleForClass:[self class]]`.  This will allow us to have the assets in the app bundle for apps that use static libraries, such as Cordova apps, and in the framework bundle for apps that utilize the framework.